include/swift/AST/DiagnosticsSema.def

```diff
@@ -2703,8 +2703,8 @@ NOTE(decl_import_via_here,none,
 NOTE(decl_import_via_local,none,
      "%kind0 is imported by this file as "
      "'%select{private|fileprivate|internal|package|%ERROR|%ERROR}1' "
-     "from %2",
-     (const Decl *, AccessLevel, const ModuleDecl*))
+     "from %select{%2|bridging header}3",
+     (const Decl *, AccessLevel, const ModuleDecl*, bool))
```

Comment:
A similar precision would be a nice to have for the exportability diagnostics referencing `DisallowedOriginKind`.